app/javascript/packs/brave_rewards_banner_display.jsx

```diff
@@ -68,6 +68,21 @@ class BraveRewardsPageForm extends React.Component {
     }
   }
 
+  backgroundFade() {
+    if(this.props.editMode){
```

Comment:
Looks good, would you fix the indentation?

Reply:
Fixed